.github/workflows/make.yml

```diff
@@ -40,7 +40,13 @@ jobs:
       run: docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }} docker.pkg.github.com
 
     - name: Build and run examples (tests)
-      run: make all
+      if: github.ref != 'refs/heads/main'
```

Comment:
I think you meant to drop this line. Don't we want to build on all branches, including main so that we can publish the artifact?

Reply:
You're right! Good catch.